### ISSUE TITLE
Dynamic unloading of Stripe

### ISSUE THREAD

**Issue by mlsad3** [ozluy/react-stripe-script-loader#3]
Way cool @ozluy ! One thing this is missing is unloading all the tracking/metrics iframes that Stripe sneaks into the website. Not 'necessary' but it would be nice to have.

For instance, take a look at what @brndnmtthws created:
https://gist.github.com/brndnmtthws/c5b28592c321533b8aa6971212295045
(Link was originally from the conversation here: https://github.com/stripe/react-stripe-elements/issues/99#issuecomment-522045812)

**Comment by ozluy** [comment]
Hi @mlsad3  I am sorry for the late response, 
- I will take a look what you mentioned above
- I will update the package if necessary
- then close the issue

Thank you for notice

**Comment by ozluy** [comment]
I created this library to avoid: 
- hardcoded Stripe script
- loading Stripe script again 

when I am routing through a Single Page Application I load Stripe script only when payment page visited and I do it only once. So I decided to keep it as it is. Removing Stripe script and iframes will lead loading one more time on a SPA. Hope I am clear enough and solution makes you happy.

**Comment by mlsad3** [comment]
no worries, and I am very happy with it :)
